api-tests/src/steps/cri-steps.ts

```diff
@@ -289,5 +290,9 @@ When(
       this.userId,
       generatePostVcsBody(credentials),
     );
+    for (const credential of credentials) {
+      await cimitStubClient.postDetectCi({ signed_jwt: credential });
+    }
+    await cimitStubClient.postMitigateCi({ signed_jwts: credentials });
```

Comment:
Would this actually work? The CIMIT stub doesn't have any mitigation rules, it has to be primed by the CRI stub...

I suppose it doesn't hurt, but potentially misleading!

Reply:
That's a good point. I was just following what core does, but it's not neccessary and is just introducing an extra API call to the tests for no reason. In future, if we wanted to arrange some state to have already mitigated CIs we could just use the management endpoint. I'll yank this.